test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+
+const server = http.Server(function(req, res) {
+  console.log('Server accepted request.');
+  res.write('Part of my res.');
+  res.destroy();
+});
+
+const requiredCallback = common.mustCall(function() {
+  console.log('Response aborted.');
+});
+
+server.listen(0, function() {
```

Comment:
`common.PORT` has the downside of making subsequent tests fail with `EADDRINUSE` if one test does not clean up after itself (due to failure, programming error, some kind of OS flakiness, or whatever). There was a bit of an effort for a while to stamp it out except in the few places that absolutely need it. I think @mscdex was at the center of that, but I may be mis-remembering.

Anyway, in general, I prefer to see `server.listen(0, ...)` over `server.listen(common.PORT, ...)` if at all possible.